### PR TITLE
V22F-494 Clean CMake

### PR DESCRIPTION
**Pull request by KungFuDonkey**

### CODE REVIEW

File: speed-dreams/source-2.2.3/cmake/FindMYSQL.cmake [hunk 1]
```diff
@@ -12,57 +12,42 @@ FIND_PATH(MYSQL_INCLUDE_DIR mysql/jdbc.h
         /..
         ${SDEXT_CUSTOM_3DPARTY_DIR})
 
-
-MESSAGE(STATUS "${MYSQL_INCLUDE_DIR}")
```
Comment (KungFuDonkey) [inline speed-dreams/source-2.2.3/cmake/FindMYSQL.cmake:16]: These were made by me, I forgot to remove them

File: speed-dreams/source-2.2.3/cmake/checks.cmake [hunk 2]
```diff
@@ -104,38 +116,33 @@ MACRO(CHECK_MYSQL_LIBRARIES)
...
-			MESSAGE(STATUS "Looking for MYSQL library cppconn8 - found")
+			IF(OPTION_SD_CMAKE_FIND_LOGS)
+				MESSAGE(STATUS "Looking for MYSQL library cppconn8 - found")
+			ENDIF(OPTION_SD_CMAKE_FIND_LOGS)
 		ELSE(MYSQL_CPPCONN8_LIBRARY)
 			MESSAGE(FATAL_ERROR "Looking for MYSQL library cppconn8 - NOT found")
 		ENDIF(MYSQL_CPPCONN8_LIBRARY)
 
-# statics are currently not used
-#		IF(MYSQL_CPPCONN_STATIC_LIBRARY)
```
Comment (KungFuDonkey) [inline speed-dreams/source-2.2.3/cmake/checks.cmake:125]: Made by me, also forgot to remove them

**Review by larsdekwant — APPROVED** [review]

File: speed-dreams/source-2.2.3/CMakeLists.txt [hunk 3]
```diff
@@ -1,6 +1,11 @@
 CMAKE_MINIMUM_REQUIRED(VERSION 3.15 FATAL_ERROR)
 
-MESSAGE(STATUS CMAKE_VERSION = "${CMAKE_VERSION}")
+OPTION(OPTION_SD_CMAKE_BUILD_LOGS FALSE)
+OPTION(OPTION_SD_CMAKE_FIND_LOGS TRUE)
```
Comment (suppergerrie2) [inline speed-dreams/source-2.2.3/CMakeLists.txt:4]: I think you should use ON since it seems to default to OFF for me

**Review by suppergerrie2 — CHANGES_REQUESTED** [review]
It defaults to OFF

**Review by suppergerrie2 — APPROVED** [review]

**Pull request merged**